From 9877bf26df93491295459aaad32f6a15b56e08ae Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 3 Mar 2015 12:43:17 +0100
Subject: [PATCH] ifcfg/write-ifcfg.sh: if a lease file is found,
 BOOTPROTO=dhcp

For all dhcp interfaces, regardless if the root mount was done via this
interface, write out the correct ifcfg file
---
 modules.d/45ifcfg/write-ifcfg.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/45ifcfg/write-ifcfg.sh b/modules.d/45ifcfg/write-ifcfg.sh
index 5e570df..e05d313 100644
--- a/modules.d/45ifcfg/write-ifcfg.sh
+++ b/modules.d/45ifcfg/write-ifcfg.sh
@@ -44,7 +44,7 @@ for netif in $IFACES ; do
         echo "DEVICE=$netif"
         echo "ONBOOT=yes"
         echo "NETBOOT=yes"
-        if [ -f /tmp/net.$netif.lease ]; then
+        if [ -f /tmp/dhclient.$netif.lease ]; then
             strstr "$ip" '*:*:*' && echo "DHCPV6C=yes"
             echo "BOOTPROTO=dhcp"
         else
